src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotationss
+        type: array
+        items:
+            type: object
+            properties:
+                uri:
```

Comment:
It will most certainly look unique. Here is an example:

```
https://hyp.is/36GLapojEeektWtTtv2WnA/elifesciences.org/inside-elife/77dd2808/taking-a-closer-look-screening-of-images
```
The hypothesis annotation id is contained within the url `36GLapojEeektWtTtv2WnA`.

Reply:
Hmm, the slug shouldn't be part of the ID as renaming the item causes it to change. Can we declare our ID, which would be `blog-article/77dd2808` in this case?

Reply:
I agree. I have not introduced an id, I was unclear whether it was necessary since I am not exposing an endpoint to retrieve a single annotation.

Reply:
Sounds like we don't want to use their URI at all, instead we can change this to an ID property.